docs/javascript-installation.md

```diff
@@ -0,0 +1,115 @@
+---
+id: javascript-installation
+title: Scope Javascript Agent installation
+sidebar_label: Installation
+---
+
+Installation of the Scope Agent is done via [npm](https://www.npmjs.com/package/@undefinedlabs/scope-agent).
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--npm-->
+
+```bash
+npm install --save-dev @undefinedlabs/scope-agent
+```
+
+<!--yarn-->
+
+```bash
+yarn add --dev @undefinedlabs/scope-agent
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Instrumenting your tests
+
+### Jest tests
+
+If you want to instrument tests run by Jest, you need to create a `reporter`.
+
+In `src/reporter.js`
+
+```javascript
+const Reporter = require("@undefinedlabs/scope-agent").JestReporter;
+
+module.exports = Reporter;
+```
+
+Then you have to configure [jest reporter](https://jestjs.io/docs/en/configuration#reporters-array-modulename-modulename-options)
+to point to this file.
+
+For example:
+
+In your `jest.config.js`:
+
+```javascript
+module.exports = {
+  // ...
+  reporters: ["<rootDir>/reporter.js"]
```

Comment:
You are using `src/reporter.js` as example path, but now you're using `<rootDir>/reporter.js`. It should be src/reporter.js again, right?

Reply:
you need to specify `<rootDir>` in `jest.config.js` https://jestjs.io/docs/en/configuration#rootdir-string. But there was indeed a mistake. It should be `<rootDir>/src/reporter.js` to keep consistency